src/main/java/org/mapstruct/intellij/expression/JavaExpressionInjector.java

```diff
@@ -177,7 +177,11 @@ public void getLanguagesToInject(@NotNull MultiHostRegistrar registrar, @NotNull
                         if ( references.length > 0 ) {
                             PsiElement resolved = references[0].resolve();
                             if ( resolved instanceof PsiMethod ) {
-                                targetType = ( (PsiMethod) resolved ).getParameterList().getParameters()[0].getType();
+                                PsiParameter[] psiParameters =
+                                        ((PsiMethod) resolved).getParameterList().getParameters();
+                                if ( psiParameters.length > 0) {
+                                    targetType = psiParameters[0].getType();
+                                }
```

Comment:
I think that we can just return `null` if the parameters are 0 here. What do you think?

Reply:
Currently this code to find the `targetType` is inside the `getLanguagesToInject` method. So we could abort the method inside the loop, but I think it is more clear to extract the code into it's own method and than use the direct return.

Reply:
You are right. The `targetType` will never be set anyways and it will be `null`. I missed that